BlogTemplate/Pages/Post.cshtml

```diff
@@ -27,6 +31,8 @@
                 <p name="Comment-Body" class="commentBody">@comment.AuthorEmail</p>
                 <p name="Comment-Date" class="commentDate">@comment.PubDate.ToString("MMM dd, yyyy")</p>
                 <p name="Comment-Body" class="commentBody">@comment.Body</p>
+                <a href="/post/@Model.Post.Slug" class="subtleLink">Reply</a>
```

Comment:
The OneNote doc specified that there should be a story to reply to another user's comment. This isn't a high priority feature, though. Should I delete the Reply links for now?